source/_integrations/kira.markdown

```diff
@@ -83,7 +87,11 @@ remotes:
 
 If no sensors or remotes are specified, a sensor with default values will be added.
 
-### Code Configuration
+### Configuration
```

Comment:
So configuration is present here again?

Reply:
I have changed the title to 'entities' to match the section content